src/bivariate_normals.jl

```diff
@@ -18,9 +20,9 @@ immutable BvnComponent{NumType <: Number}
     z::NumType
     dsiginv_dsig::Matrix{NumType}
 
-    function BvnComponent{T1 <: Number, T2 <: Number, T3 <: Number}(
+    BvnComponent{T1 <: Number, T2 <: Number, T3 <: Number}(
         the_mean::Vector{T1}, the_cov::Matrix{T2}, weight::T3;
-        calculate_siginv_deriv::Bool=true)
+        calculate_siginv_deriv::Bool=true) = begin
```

Comment:
Let's use `function` syntax everywhere for functions rather than begin/end blocks.